src/components/DraggableList/useDraggableInPortal.ts

```diff
@@ -0,0 +1,43 @@
+import type {DraggableChildrenFn} from 'react-beautiful-dnd';
+import {useEffect, useRef} from 'react';
+import {createPortal} from 'react-dom';
+
+type DraggableInPortalProps = {
+    shouldUsePortal?: boolean;
+};
+
+export default function useDraggableInPortal({shouldUsePortal}: DraggableInPortalProps): (render: DraggableChildrenFn) => DraggableChildrenFn {
```

Comment:
Can you rename the function not to use `use` at the front?

Maybe `createDraggableInPortal`? `useSomething` is used for hook names